src/components/form/button.js

```diff
@@ -82,6 +83,26 @@ export const SharedButtonStyle = css`
                     color: var(--color-red);
                 }
             `
+        if (props.hero)
+            return css`
+                border: 2px solid var(--color-red);
```

Comment:
`Border` is accounted for the height of the element for the top and bottom as well, so if we want a 64px button, you need to either use `box-shadow` instead or minus 2px from the vertical padding, then we should be able to remove the height prop.

Reply:
I've removed the height props and added the `box-shadow` to replace border.